mcs/class/System.ServiceModel/Test/System.ServiceModel.Dispatcher/DispatchRuntimeTest.cs

```diff
@@ -231,6 +232,20 @@ public void TestClientRuntime ()
 			r.ClientOperations.CopyTo(operations, 0);
 			Assert.AreEqual (operations[0], operation1, "operations #7");
 			Assert.AreEqual (operations[1], operation2, "operations #8");
+
+			Assert.AreEqual (r.UnhandledClientOperation.Name, "*", "unhandled #1");
+			Assert.AreEqual (r.UnhandledClientOperation.Action, "*", "unhandled #2");
+			Assert.AreEqual (r.UnhandledClientOperation.ReplyAction, "*", "unhandled #3");
+			Assert.AreEqual (r.UnhandledClientOperation.Parent, r, "unhandled #4");
+			Assert.IsNotNull (r.UnhandledClientOperation.Formatter, "unhandled #5");
+			Assert.IsFalse (r.UnhandledClientOperation.DeserializeReply, "unhandled #6");
+			Assert.IsFalse (r.UnhandledClientOperation.SerializeRequest, "unhandled #7");
+			Assert.IsNull (r.UnhandledClientOperation.BeginMethod, "unhandled #8");
+			Assert.IsNull (r.UnhandledClientOperation.EndMethod, "unhandled #9");
+			Assert.IsNull (r.UnhandledClientOperation.SyncMethod, "unhandled #10");
+			Assert.AreEqual (r.UnhandledClientOperation.ParameterInspectors.Count, 0, "unhandled #11");
+			Assert.AreEqual (r.UnhandledClientOperation.ClientParameterInspectors.Count, 0, "unhandled #12");
+			Assert.AreEqual (r.UnhandledClientOperation.FaultContractInfos.Count, 0, "unhandled #13");
```

Comment:
Seems like you've got the expected and actual values reversed here?

Reply:
Thanks for spotting it, it's been updated.